src/mjolnir/ferry_connections.cc

```diff
@@ -6,6 +6,8 @@
 #include "baldr/graphconstants.h"
 #include "midgard/util.h"
 
+#define UNUSED(x) (void)(x)
```

Comment:
damn, I forgot to removed unused "unused" macro.. I'll clean it up in the next PR